lib/applications/schema.graphql

```diff
@@ -0,0 +1,114 @@
+type Application {
+  # Applicant information
+  id: ID!
+  firstName: String!
+  middleName: String!
+  lastName: String!
+  dateOfBirth: Date!
+  gender: Gender!
+  email: String!
+  phone: String!
+  province: Province!
+  city: String!
+  address: String!
+  postalCode: String!
+  notes: String
+  rcdUserId: Int
+  isRenewal: Boolean!
+  poaFormUrl: String
+  applicantId: Int
+  
+  # Medical information
+  disability: String!
+  affectsMobility: Boolean!
+  mobilityAidRequired: Boolean!
+  cannotWalk100m: Boolean!
+  aid: [Aid!]
```

Comment:
Since `aid` is non-null in our DB, we should have: 
```suggestion
  aid: [Aid!]!
```

Same for the create mutation input.